File: 0001-two-sum/0001-two-sum.go
func twoSum(nums []int, target int) []int {
    
    tmpmap := make(map[int]int)
    
    for i:= 0; i < len(nums); i++ {
        b := target - nums[i]
        value, exist := tmpmap[b]
        if exist {
            return [] int{value, i}
        } else {
            tmpmap[nums[i]] = i
        }
    }
    return nil
    
}

File: 0001-two-sum/0001-two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        map1 = {}
        for i in range(len(nums)):
            b = target - nums[i]
            if map1.get(b) != None:
                return [map1.get(b), i]
            else:
                map1[nums[i]] = i
        return